### ISSUE TITLE
filter strand invasion

### ISSUE THREAD

**Issue by Lixinyoung** [StatBiomed/CamoTSS#4]
Hello!
  Very important tool. When I check your code, I think the first step to filter the strand invasion in function`_getreads`, you should take strandness into consideration, just as your second step(SCAFE method) to remove strand invasion. Like following:  

```python
if generefdf.loc[geneid]['Strand']=='+':
    reads1_umi=[r for r in reads1_umi if editdistance.eval(fastqFile.fetch(start=r.reference_start-14, end=r.reference_start-1, region='chr'+str(self.generefdf.loc[geneid]['Chromosome'])),'TTTCTTATATGGG') >3 ]
elif generefdf.loc[geneid]['Strand']=='-':
    reads1_umi=[r for r in reads1_umi if editdistance.eval(fastqFile.fetch(start=r.reference_end, end=r.reference_end+13, region='chr'+str(self.generefdf.loc[geneid]['Chromosome'])),'CCCATATAAGAAA') >3 ]
```

**Comment by Lixinyoung** [comment]
By the way, I wonder how can I use Squence Features(Convolution Model) to filter false positive clusters? I didn't find the corresponding parameters.

**Comment by houruiyan** [comment]
Yes, thank you very much to point out this error! I agree with you! 
For the sequence features, we did not embed it to the CamoTSS for convenient. Because this feature model has already been good performance! But we have the pretrained CNN model on the PBMC dataset. I will submit it to the model folder later. 
Thanks!
Ruiyan

**Comment by Lixinyoung** [comment]
Thanks for the prompt reply. I understand :)

**Comment by houruiyan** [comment]
Hi xinyoung,

I submit the pretrained sequence model to the CamoTSS/model folder and also attach the code use this pretrained model on test data in the notebook/use_pretrained_sequence.py. 

Best regards,
Ruiyan
